tests/pinkopy/test_helper.py

```diff
@@ -0,0 +1,52 @@
+import requests_mock
+import uuid
+
+
+def get_uuid():
+    return str(uuid.uuid4())
+
+
+def mock_session(base_session, service='http://' + get_uuid(),
+                 user=get_uuid(), pw=get_uuid(), token=get_uuid(),
+                 content_type='application/json', clients=get_uuid()
+                 ):
```

Comment:
I'll change these to what you have suggested. I grabbed UUIDs to make sure that the correct values were being validated but what you have would accomplish the same purpose.